### ISSUE TITLE
Cannot start games

### ISSUE THREAD

**Issue by chrispy212** [LibreVR/Revive#1467]
**Describe the bug**
Trying to launch a game from the revive dashboard does nothing

**Environment (please complete the following information):**
 - App: Vader Immortal
 - OS: Win10
 - Headset: Vive

**Versions (please complete the following information):**
 - Revive: 2.0.0.13
 - SteamVR: 1.9.16

**Additional context**
Log as attached https://pastebin.com/03yJ6hTQ

**Comment by CrossVR** [comment]
> Failed to add manifest file to OpenVR: C:\Program Files\Revive\revive.vrmanifest (VRApplicationError_InvalidManifest)

Please post the file mentioned in the log

**Comment by chrispy212** [comment]
Weirdly when I've gone through the reinstalls that error has disappeared. I've attached a newer log.

**Comment by CrossVR** [comment]
Looks like a successful launch now

**Comment by chrispy212** [comment]
Yes, but it isn't. The game still doesn't launch. A black screen appears, then hangs forever.

**Comment by CrossVR** [comment]
Sound like issue #1392

**Comment by chrispy212** [comment]
People with #1392 seem to mention that elements of the game are in fact loading; audio snippets, lightsaber models etc. What I'm getting it the Vive drops me into the grid (i.e. leaves SteamVr Home as though the game is loading), says "Vader Immortal - Up Next" in the little card, and the monitor screen goes black. This is as far as I get. The black screen seems to stop responding after loading ~240mb into RAM.
